src/Orchard.Web/Modules/Orchard.MediaLibrary.WebSearch/Module.txt

```diff
@@ -0,0 +1,23 @@
+﻿Name: Orchard.MediaLibrary.WebSearch
+AntiForgery: enabled
+Author: The Orchard Team
+Website: https://orchardproject.net
+Version: 1.0
+OrchardVersion: 1.10.3
+Description: Enables Orchard to use image search in media or as a provider.
+Features:
+    Orchard.MediaLibrary.WebSearch:
+        Description: Enables Orchard to use image search in media or as a provider.
```

Comment:
Base functionality for Orchard to be able import images from web search results to the Media Library.